test/skate/migrate_test.exs

```diff
@@ -0,0 +1,119 @@
+defmodule Skate.MigrateTest do
+  use ExUnit.Case
+  import Test.Support.Helpers
+  import ExUnit.CaptureLog
+
+  describe "start_link/1" do
```

Comment:
**suggestion:** for the async migrations the logging is tested in the `handle_continue/1` tests, but there are no tests for the logging for sync migrations - maybe that could be added here, or by testing `init/1` directly?

Reply:
Good idea! I'll add those. For some context on why the async logging wasn't tested directly in the regular tests - there seemed to be some race conditions with `capture_log` not capturing all of the logs from the async step because `start_link` had already returned.

Reply:
Ah, yup, makes sense.